app/views/campaign_pages/new.slim

```diff
@@ -1,4 +1,3 @@
-.jumbotron
-  h1 Create a new page
 
 = render 'form'
+
```

Comment:
The `new` view of the campaign_pages_controller is broken on the current branch because it references `@options`, which has things like all the languages and the campaigns, and has been removed from the controller. I'm confused how page creation happened in the demo, and what should become of this view.